### PR TITLE
Translate the keepalive / holdtime from api to frr

### PR DESCRIPTION
**Pull request by fedepaol**
Those fields were missing. Also, we do validate their values as we did in metallb.

Note: this requires https://github.com/metallb/metallb/pull/2104 to make the tests compile

### CODE REVIEW

File: internal/controller/api_to_config_test.go [hunk 1]
```diff
@@ -295,7 +305,8 @@ func TestConversion(t *testing.T) {
 								Name:     "65031@192.0.2.16",
 								ASN:      65031,
 								Addr:     "192.0.2.16",
-								Port:     179,
+
```
Comment (oribon) [inline internal/controller/api_to_config_test.go]: nit: unrelated

File: e2etests/tests/session.go [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+								ASN: infra.FRRK8sASN,
+								Neighbors: []frrk8sv1beta1.Neighbor{
+									{
+										ASN:     1234,
+										Address: "192.168.1.1",
+										HoldTime: metav1.Duration{
+											Duration: 90 * time.Second,
+										},
+										KeepaliveTime: metav1.Duration{
+											Duration: 30 * time.Second,
```
Comment (oribon) [inline e2etests/tests/session.go:82]: is there value in using something other than the defaults?

**Review by oribon — APPROVED** [review]
small nits, lgtm

File: e2etests/tests/session.go (hunk 2 above)
Comment (fedepaol) [reply e2etests/tests/session.go]: there is, as by creating with the api doesn't use the CRD default values. However, slightly changing the values so it offers better coverage

File: internal/controller/api_to_config_test.go (hunk 1 above)
Comment (fedepaol) [reply internal/controller/api_to_config_test.go]: fixed
